static/images/unknown-user-avatar-medium.png

```diff
@@ -0,0 +1 @@
+unknown-user-avatar.png
```

Comment:
Is it easy to add a test that accessing the `-medium.png` URL works?

Reply:
I cannot think of any way to test static file URLs.